.github/ISSUE_TEMPLATE/documentation.md

```diff
@@ -0,0 +1,11 @@
+---
+name: Documentation
+about: Improve an existing feature or workflow
+labels: 'docs'
+---
+
+### What do you want to improve?
+
+### What is the current documentation?
+
+### What is the new documentation?
```

Comment:
Not sure these are needed.

Reply:
In case we miss some doc in kiali.io ...it would be great